### PR TITLE
Update .spec file

### PR DESCRIPTION
**Pull request by redhatrises**
- Update .spec file to handle both python2 and python3 for RHEL and Fedora

### CODE REVIEW

**Comment by dahaic** [comment]
Jenkins test this please.

File: oscap-anaconda-addon.spec [hunk 1]
```diff
@@ -48,9 +63,12 @@ make install DESTDIR=%{buildroot}
 %files -f %{name}.lang
 %{_datadir}/anaconda/addons/org_fedora_oscap
 
-%doc COPYING ChangeLog README
+%doc COPYING ChangeLog README.md
```
Comment (dahaic) [inline oscap-anaconda-addon.spec]: Update `Makefile` to contain `README.md` instead of `README`. This should fix the issue jenkins reports.

Comment (redhatrises) [reply oscap-anaconda-addon.spec]: @dahaic done

**Comment by dahaic** [comment]
Jenkins test this please.

**Comment by dahaic** [comment]
Jenkins test this please.

**Comment by dahaic** [comment]
Fedora failures are expected, should get fixed with update to `openscap-python3`. ACK

**Comment by dahaic** [comment]
Great work @redhatrises on getting this mess cleaned up!!

**Pull request merged**